layerInfo.json

```diff
@@ -151,6 +151,17 @@
 					"@fluid-tools/webpack-fluid-loader"
 				]
 			},
+			"ServiceClients": {
+				"dirs": ["packages/service-clients/"],
+				"deps": [
+					"Container-Definitions",
+					"Driver",
+					"Framework",
+					"Runtime",
+					"Routerlicious-Driver",
+					"Server-Shared-Utils"
+				]
+			},
 			"AzureClient": {
 				"dirs": ["azure/packages/"],
 				"deps": [
```

Comment:
not sure if this should be removed. `azure` directory has other packages such as azure-local-service, azure-service-utils and e2e test. 

I'm planning to move azure e2e test to service-clients directory in next follow-up PR.

Reply:
Definitely fine to defer for now, but longer term I could imagine getting rid of the azure directory entirely

Reply:
will azure-local-service and azure-service-utils live in service-clients?